modules/ROOT/partials/azb-credentials.adoc

```diff
@@ -1,21 +1,23 @@
-You need an Azure account, an Azure storage account and blob container.
-
-We use the link:https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredential?view=azure-java-stable[default Azure credentials] to authenticate the neo4j or neo4j-admin process against Azure.
-You can check their documentation to see the different authentication methods supported.
-
-The simplest method is to use the link:https://learn.microsoft.com/en-us/cli/azure/install-azure-cli[azure cli tool]
-
-Execute this, and follow the steps to authenticate.
-
+. Ensure you have a Azure account, an Azure storage account, and a blob container.
+.. You can create a storage account using the Azure portal. +
+For more information, see the Azure official documentation on link:https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal[Create a storage account].
+.. Create a blob container in the Azure portal. +
+For more information, see the Azure official documentation on link:https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal[Quickstart: Upload, download, and list blobs with the Azure portal].
+. Install the Azure CLI by following the instructions in the Azure official documentation -- link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli[Azure official documentation].
+. Authenticate the neo4j or neo4j-admin process against Azure using the default Azure credentials. +
+See the Azure official documentation on link:https://learn.microsoft.com/en-us/java/api/com.azure.identity.defaultazurecredential?view=azure-java-stable[default Azure credentials] for more information. +
+Alternatively, you use another supported authentication method.
+See the Azure official documentation on link:https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli[Authenticate with Azure CLI] for more information.
```

Comment:
This isn't very clear. The only supported methods are the ones referenced in the previous page ("default Azure credentials"). So this isn't an alternative, it is just one of the supported methods, the one I considered the simplest to use. 

We could also remove this phrase. It isn't essential, readers will be able to find this information based on the previous link provided